lib/assessment/rubric_tester.py

```diff
@@ -302,57 +306,54 @@ def main():
-        params = get_params(experiment_lesson_prefix)
+        params = get_params(params_lesson_prefix)
         response_type = params.get('response-type', 'tsv')
-        prompt, standard_rubric = read_inputs(prompt_file, standard_rubric_file, experiment_lesson_prefix)
+        prompt, standard_rubric = read_inputs(prompt_file, standard_rubric_file, params_lesson_prefix)
         student_files = get_student_files(options.max_num_students, dataset_lesson_prefix, student_ids=options.student_ids)
         if os.path.exists(os.path.join(dataset_lesson_prefix, actual_labels_file_old)):
             actual_labels = get_actual_labels(actual_labels_file_old, dataset_lesson_prefix)
         else:
             actual_labels = get_actual_labels(actual_labels_file, dataset_lesson_prefix)
-        examples = get_examples(experiment_lesson_prefix, response_type)
+        examples = get_examples(params_lesson_prefix, response_type)
 
         validate_rubrics(actual_labels, standard_rubric)
         validate_students(student_files, actual_labels)
         rubric = standard_rubric
 
         # set up output and cache directories
-        os.makedirs(os.path.join(experiment_lesson_prefix, output_dir_name), exist_ok=True)
-        os.makedirs(os.path.join(experiment_lesson_prefix, cache_dir_name), exist_ok=True)
+        os.makedirs(os.path.join(params_lesson_prefix, output_dir_name), exist_ok=True)
+        os.makedirs(os.path.join(params_lesson_prefix, cache_dir_name), exist_ok=True)
         if not options.use_cached:
-            for file in glob.glob(f'{os.path.join(experiment_lesson_prefix, cache_dir_name)}/*'):
+            for file in glob.glob(f'{os.path.join(params_lesson_prefix, cache_dir_name)}/*'):
                 os.remove(file)
 
         # call label function to either call openAI or read from cache
         with concurrent.futures.ThreadPoolExecutor(max_workers=options.workers) as executor:
-            predicted_labels = list(executor.map(lambda student_file: read_and_label_student_work(prompt, rubric, student_file, examples, options, params, experiment_lesson_prefix, response_type), student_files))
+            predicted_labels = list(executor.map(lambda student_file: read_and_label_student_work(prompt, rubric, student_file, examples, options, params, params_lesson_prefix, response_type), student_files))
 
         errors = [student_id for student_id, labels in predicted_labels if not labels]
         # predicted_labels contains metadata and data (labels), we care about the data key
         predicted_labels = {student_id: labels['data'] for student_id, labels in predicted_labels if labels}
 
         for is_pass_fail in [True, False]:
             output_filename = 'report-pass-fail.html' if is_pass_fail else 'report-exact-match.html'
-            output_file = os.path.join(experiment_lesson_prefix, output_dir_name, output_filename)
+            output_file = os.path.join(params_lesson_prefix, output_dir_name, output_filename)
```

Comment:
re: our conversation in check-in about using the reports in our process- I think an arg to optionally add a user-defined string/ maybe a timestamp to the output file names would be a good idea to differentiate between outputs during development (ofc adding these to the .gitignore in the release data repo). A current output file for each lesson that should be updated in a PR might also probably be a good artifact for the data repo

Reply:
probably not necessary before merging this, but would be a nice follow-up

Reply:
Yeah, I agree we need to clarify this. For now, I'll leave the output in here, and we can always move it later.